kits/nl-to-sql-agent/apps/.env.example

```
@@ -0,0 +1,25 @@
+NL_TO_SQL_FLOW_ID=your_deployed_flow_id
+
+# Session secret for iron-session
+SESSION_PASSWORD=a_complex_password_at_least_32_chars_long!!
+
+# Demo authentication configuration
+# Set DEMO_AUTH_ENABLED to "true" to enable demo authentication.
+# Both DEMO_USERNAME and DEMO_PASSWORD must be configured when enabled.
+# Demo authentication is disabled by default for security (fail-closed).
+DEMO_AUTH_ENABLED=true
+DEMO_USERNAME=your-demo-username
+DEMO_PASSWORD=your-demo-password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEMO_PASSWORD key should go before the DEMO_USERNAME key

(UnorderedKey)